openfisca_us/parameters/gov/states/ca/tax/income/deductions/standard/amount.yaml

```diff
@@ -0,0 +1,23 @@
+description: Amount an individual or couple can receive via standard deduction
+
+SINGLE:
+  2020-01-01: 4_803
+SEPARATE:
+  2020-01-01: 4_803
+WIDOW:
+  2020-01-01: 9_606
+JOINT:
+  2020-01-01: 9_606
+HEAD_OF_HOUSEHOLD:
+  2020-01-01: 9_606
+
+metadata:
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  name: ca_standard_deduction_amount
+  label: "California standard deduction amount"
```

Comment:
BTW, we don't need quotes unless there's a symbol like `:` that confuses the YAML parser.